### DISCUSSION TITLE
DAG is not Triggered with Airflow v2.7.2

### DISCUSSION THREAD

**Discussion by smitesh-maniya** [apache/airflow#36080] [General]
When i am going to trigger DAG i am facing seeing below page. I can not see logs of DAG.
**Operating System**
Ubuntu 20.04.6 LTS
![image](https://github.com/apache/airflow/assets/123544692/8e2209ea-e927-46b7-9fde-d25c6072e6be)

**Comment by potiuk** [comment]
As instructed in the mesage - you ned to look at logs. there is no way someone can do it for you remotely to help you. You will find detailed information (and it's a good exercise to learn how to look at your logs in your deployment. That's a useful thing you should learn.
